packages/uploadthing/nuxt.ts

```diff
@@ -0,0 +1,6 @@
+import { createBuilder } from "./src/upload-builder";
+
+export { createNuxtRouteHandler } from "./src/nuxt";
+export type { FileRouter } from "./src/internal/types";
+
+export const createUploadthing = () => createBuilder<"nuxt">();
```

Comment:
is nuxt not Request/Response based?

Reply:
It's using [h3](https://github.com/unjs/h3) library under the hood which relies on node:http module. Therefore different types are being used